client/commonFramework/detect-unsafe-code-stream.js

```diff
@@ -7,6 +7,9 @@ window.common = (function(global) {
   const detectFunctionCall = /function\s*?\(|function\s+\w+\s*?\(/gi;
   const detectUnsafeJQ = /\$\s*?\(\s*?\$\s*?\)/gi;
   const detectUnsafeConsoleCall = /if\s\(null\)\sconsole\.log\(1\);/gi;
+  const detectInComment = new RegExp(['\\/\\/[\\W\\w\\s]*?function.|',
```

Comment:
So this is that regex with the proper escaping 

```js
/\/\/[\W\w\s]*?function.|\/\*[\s\w\W]*?function[\s\W\w]*?\*\//
```
But I can't get this to work properly.